### PR TITLE
Fix Prism Powder to mimic retail behavior

### PR DESCRIPTION
**Pull request by Wiggo32**
Prism Powder overwrites invisible in retail

### CODE REVIEW

File: scripts/globals/items/pinch_of_prism_powder.lua [hunk 1]
```diff
@@ -12,7 +12,8 @@ end;
 
 function onItemUse(target)
     local duration = math.random(60, 180);
-    if (not target:hasStatusEffect(EFFECT_INVISIBLE)) then
-        target:addStatusEffect(EFFECT_INVISIBLE, 0, 10, math.floor(duration * SNEAK_INVIS_DURATION_MULTIPLIER));
+    if (target:hasStatusEffect(EFFECT_INVISIBLE)) then
```
[the remark this answers is not in the document]
Comment (Wiggo32) [reply scripts/globals/items/pinch_of_prism_powder.lua]: hmmm... that didn't work. means I'd have to figure out where it is flagged in db

[the remark this answers is not in the document]
Comment (Wiggo32) [reply scripts/globals/items/pinch_of_prism_powder.lua]: i mean... well i know where it is in the db but the flags i need to figure out and decipher

[the remark this answers is not in the document]
Comment (TeoTwawki) [reply scripts/globals/items/pinch_of_prism_powder.lua]:
```cpp
enum EFFECTOVERWRITE
{
    EFFECTOVERWRITE_EQUAL_HIGHER = 0, // only overwrite if equal or higher
    EFFECTOVERWRITE_HIGHER = 1, // only overwrite if higher
    EFFECTOVERWRITE_NEVER = 2, // never overwrite
    EFFECTOVERWRITE_ALWAYS = 3, // always overwrite no matter
    EFFECTOVERWRITE_IGNORE = 4 // ignore dupes
};
```
are the effect flag values

[the remark this answers is not in the document]
Comment (TeoTwawki) [reply scripts/globals/items/pinch_of_prism_powder.lua]:
```sql
|INSERT INTO `status_effects` VALUES (69,'invisible',3429,0,0,2,0,0,4,0);
```
is what we have in the sql so its set to 2. I dunno why its 2.
OH, prolly to keep jig form replacing spell invis? I don't remember what retail behavior is there. I wanna say it oughta be 0 or 3..

[the remark this answers is not in the document]
Comment (Wiggo32) [reply scripts/globals/items/pinch_of_prism_powder.lua]: prism powder in db 'flags' = 1540. I don't know if we should change that number or add/subtract to it.

[the remark this answers is not in the document]
Comment (TeoTwawki) [reply scripts/globals/items/pinch_of_prism_powder.lua]: wrong column and my fau;t for calling it "flags" (I edited my other comment)

**Pull request merged**